static/examples/6.x/hidden-components.js

```diff
@@ -25,12 +25,16 @@ export default function App() {
         screenOptions={{ headerShown: false }}
       >
         <Stack.Screen name="Home">
-          {() => (
-            <Tab.Navigator initialRouteName="Analitics" tabBar={() => null}>
-              <Tab.Screen name="Analitics" component={Demo} />
-              <Tab.Screen name="Profile" component={Demo} />
-            </Tab.Navigator>
-          )}
+        {() => (
+          <Tab.Navigator
+            initialRouteName="Analitics"
+            tabBar={() => null}
```

Comment:
It is like that [since v5.](https://github.com/react-navigation/react-navigation.github.io/blob/main/static/examples/5.x/hidden-components.js#L28) 
[This particular example](https://reactnavigation.org/docs/handling-safe-area#hiddencustom-header-or-tab-bar) shows how to deal with safe areas and that's why tabs are hidden. I've also hidden the header to match what the example was trying to show.

Reply:
@kacperkapusciak oki, I think we need to revisit that page, currently, I see 2 issues:
- it seems pointless to use a tab navigator if you're going to hide the tab bar
- the title mentions "custom header", but you don't need to add a safe area to your screen because you use a custom header
- this example is not typical, there are often screens with header and no tab bar (not hidden)

what it should document is
- general guide to using safe area on any screen that touches any screen edges - even if it has both header and tab bar, it's still necessary to handle horizontal insets (can be merged with landscape mode)
- mention that you need top safe area when not using a header/hiding header/transparent header
- mention that you need bottom safe area when not using a tab bar/transparent tab bar
- we should document an example for `ScrollViews` and `FlatList` because they are common to be in screens

I'll create a task on notion for that